pkg/localmetrics/localmetrics.go

```diff
@@ -21,8 +21,10 @@ import (
 
 var log = logf.Log.WithName("network-attachment-definition")
 var (
-	netAttachDefInstanceEnabledCount      = 0.0
-	netAttachDefInstanceSriovEnabledCount = 0.0
+	netAttachDefInstanceEnabledCount      = 0
+	netAttachDefInstanceSriovEnabledCount = 0
+	//Change this when we set metrics per node.
+	objStore = make(map[string]string, 330) // Preallocate room 110 entires per node*3
```

Comment:
Aneesh maybe we could define constants for these in case of needing to change in the future?